Main.java

```diff
@@ -0,0 +1,62 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Scanner reader = new Scanner(System.in);
+
+        Circle myCircle = new Circle(0);
```

Comment:
You don't need these three objects initialised here since they will technically not be used later